createSpotifyPlaylist.py

```diff
@@ -0,0 +1,244 @@
+"""
+This module is used to create spotify playlist from other playlists. This is possible by using the
+spotify api which allows us to search and retrive data. We use the spotify playlists to search for
+playlist with a provided search query. All the playlists that are found will be search for its most
+popular songs. Those songs will then be added to a new spotify playlist which will be craeted here.
+This module could be run on its own to create playlist for whoever wants it or could also be run
+using the module `twitter.py` thats in this project. What this will do is check the users twitter
+timeline and create playlists by checking @mentions and using those as search querys. This will
+require both the spotify and twitter api.
+"""
+import os
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+
+
+class CreateSpotifyPlaylist:
+    """
+    A class used to represent an Animal
+
+    ...
```

Comment:
fill this in - clearly this is WIP still

Reply:
I have added a class docstring